Resources/Prototypes/Species/diona.yml

```diff
@@ -0,0 +1,118 @@
+- type: species
+  id: Diona
+  name: species-name-diona
+  roundStart: true
+  prototype: MobDiona
+  sprites: MobDionaSprites
+  markingLimits: MobDionaMarkingLimits
+  dollPrototype: MobDionaDummy
+  skinColoration: TintedHues
+
```

Comment:
Is there a nameset floating around somewhere you could use? Maybe check whichever BYOND codebase you ported them from?